odl/phantom/geometric.py

```diff
@@ -90,22 +91,26 @@ def phantom(x):
     return space.element(phantom)
 
 
-def defrise(space, nellipses=8, alternating=False):
+def defrise(space, nellipses=8, alternating=False, min_pt=None, max_pt=None):
     """Phantom with regularily spaced ellipses.
 
     This phantom is often used to verify cone-beam algorithms.
 
     Parameters
     ----------
-    space : `DiscretizedSpace`
-        Discretized space in which the phantom is supposed to be created.
-        Needs to be 2d or 3d.
+    space : `DiscreteLp`
+        Space in which the phantom should be created, must be 2- or
+        3-dimensional.
     nellipses : int, optional
         Number of ellipses. If more ellipses are used, each ellipse becomes
         thinner.
     alternating : bool, optional
         True if the ellipses should have alternating densities (+1, -1),
         otherwise all ellipses have value +1.
+    min_pt, max_pt : array-like, optional
+        If provided, use these vectors to determine the bounding box of the
+        phantom instead of ``space.min_pt`` and ``space.max_pt``. Providing
+        one of them results in a shift, and providing both in a scaling.
```

Comment:
This needs further explanation

Reply:
What is unclear?

Reply:
Added a bit of text.

Reply:
What do we mean by " a shift", where does it go? In what units? Perhaps by example is best here.

Reply:
Example added